docs: Updated Tree and List docs (#2241)
looker-open-source/components: www/src/documentation/components/content/tree.mdx

</>
```

## Changing Background Behavior

By default, `Tree`s and `TreeItem`s background color from being hovered (or other state changes like the `selected` and `current` props) will start left of their indent padding.

Use `Tree`'s `labelBackgroundOnly` prop to remove the indent padding from being included in the backgrounds of both `Tree` and `TreeItem`.

> Because of the layout changes from `labelBackgroundOnly`, a separate callback must be assigned to the indent spacing of child `TreeItem`s. Use `TreeItem`'s `onClickWhitespace` prop to assign a callback to the indent padding of a given item.

Reviewer: ```suggestion
The layout changes from `labelBackgroundOnly`, necessitate a separate callback be assigned to the indent spacing of child `TreeItem`s. Use `TreeItem`'s `onClickWhitespace` prop to assign a callback to the indent padding of a given item. Generally the callback specified on `onClickWhitespace` is the same as specified on the primary item within the `ListItem`.
```

**Note:** When `labelBackgroundOnly` is true, `TreeItem`s should use `itemRole="none"`.

```jsx
<Tree label="Cheeses" labelBackgroundOnly defaultOpen>
  <TreeItem itemRole="none" onClickWhitespace={() => alert('Gouda time.')}>

>     Gouda

Reviewer: ```suggestion
    <span onClick={() => alert('Gouda time.')}>Gouda</span>
```

  </TreeItem>
  <TreeItem
    itemRole="none"
    onClickWhitespace={() => alert('Swiss time.')}
    selected
  >

>     Swiss

Reviewer: ```suggestion
    <span onClick={() => alert('Swiss time.')}>Swiss</span>
```

  </TreeItem>
  <TreeItem itemRole="none" onClickWhitespace={() => alert('Cheddar time.')}>

>     Cheddar

Reviewer: ```suggestion
    <span onClick={() => alert('Cheddar time.')}>Cheddar</span>
```
